services/optimiser/testing_for_scheduler.py

```diff
@@ -0,0 +1,49 @@
+import pytest
+from domain.base import Session, Engine
+from services.optimiser.input_processing import (
+    get_input_clashes,
+    get_vehicle_time,
+    get_vehicle_list,
+    get_position_list,
+    get_position_list_all,
+
+)
+
+
+@pytest.fixture
+def session():
+    new_session = Session(bind=Engine)
+    yield new_session
+
+    new_session.close()
+
+
+@pytest.fixture
+def request_id():
+    return '322'
+
+@pytest.fixture
+def vehicle_id():
+    return  '369'
+
+
+
+def test_get_input_clashes(session,request_id):
```

Comment:
Please add comments to all of the test cases describing what the expected output is. This will help future maintainers work out what you were actually testing.

Reply:
If you are unsure about how to write the comments, you can read my update and I hope it will help.